py/torch_tensorrt/dynamo/runtime/_TorchTensorRTModule.py

```
@@ -275,6 +293,9 @@ def set_extra_state(self, state: SerializedTorchTensorRTModuleFmt) -> None:
     def set_pre_allocated_outputs(self, enable: bool) -> None:
```

What is the relationship between this method and using output allocator

---

`use_pre_allocated_outputs` will be silently ignored for now because it is expected to use in CG mode. If OA is enabled, there's no code calling `use_pre_allocated_outputs`.

---

If its not an API intended for users consider adding a _ prefix for it like `_use_preallocated_outputs`

---

Oh sorry for the confusion. I meant `use_pre_allocated_outputs` will be ignored for OA mode. It's still useful in CG mode. 

For example: 
Given DDS `compiled_model`, the following two cases will use OA and ignore `enable_pre_allocated_outputs` context manager:
```
with torch_tensorrt.runtime.enable_pre_allocated_outputs(compiled_model):
    cg_out = compiled_model(*inputs)

or 

with torch_tensorrt.runtime.enable_output_allocator(compiled_model):
    with torch_tensorrt.runtime.enable_pre_allocated_outputs(compiled_model):
        cg_out = compiled_model(*inputs)
```

Given NonDDS `compiled_model`, it will keep the original behavior (i.e., in CG mode)